src/oph/ehoks/virkailija/system_handler.clj

```diff
@@ -196,9 +196,9 @@
       :path-params [hoks-id :- s/Int]
       (let [hoks (db-hoks/select-hoks-by-id hoks-id)]
         (if hoks
-          (if (:osaamisen-hankkimisen-tarve hoks)
+          (if (op/send? :aloituskysely hoks)
```

Comment:
Nää sais tietysti pois, jos check-and-send! / send-if-needed!  palauttais että lähtikö vai ei.